.travis.yml

```diff
@@ -1,15 +1,17 @@
 language: julia
 julia:
   - 0.6
+  - 0.7
+  - 1.0
   - nightly
 notifications:
     email: false
 sudo: false
 script:
   - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-  - julia --check-bounds=yes -e 'Pkg.clone(pwd()); Pkg.test("Coverage"; coverage=true)'
-  - julia --code-coverage=user -e 'cd(Pkg.dir("Coverage")); using Coverage; cov_res = process_folder(); Coveralls.submit(cov_res); Codecov.submit(cov_res)';
+  - julia --check-bounds=yes etc/travis-test.jl
+  - julia --code-coverage=user etc/travis-coverage.jl
```

Comment:
The `script` section should just me removed so that it uses the default script, which handles all of this.

Reply:
Awesome! So, where can I see what the "default script" is, then?

The information on https://docs.travis-ci.com/user/languages/julia/#default-build-and-test-script is rather sparse, and it would be good to know where it comes from. In particular, it does not reveal whether options like `--check-bounds=yes`  or `--code-coverage=user` are passed to Julia, and/or how to control that.

I also wondered why the coverage submission is seemingly run twice: once here, and once below in the `after_success` section?

Reply:
Thanks! (though unfortunately I didn't see your comment before, and just had found it there myself, wasting some time searching *sigh*).

So the documentation on <https://docs.travis-ci.com/user/languages/julia/#default-build-and-test-script> simply is wrong then (good, it had me worried for a bt): it claims that for new style projects, this is used:
```
using Pkg
Pkg.build()
Pkg.test()
```
But it really uses (the equivalent of)
```
using Pkg
Pkg.build()
Pkg.test(coverage=true);
```

Reply:
Yeah pretty much